tests/cypress/tests/unit/exportUtils_tests.ts

```diff
@@ -49,92 +50,92 @@ describe('export', () => {
     describe('getMergeRanges', () => {
         it('no merge', () => {
             const testedArray = [['a', 'b', 'c', 'd'],
-                                 ['a', 'b', 'c', 'd'],
-                                 ['a', 'b', 'c', 'd']];
+            ['a', 'b', 'c', 'd'],
+            ['a', 'b', 'c', 'd']];
```

Comment:
NBD but did a linter do that? Seems markedly less readable than some flavor of aligned.

Reply:
I have VSCode auto-formatting turned on, must have done that when I added the import https://github.com/plotly/dash-table/pull/637/files/28c73dd0f967b7a5ed23092a95a4f7509567d307#diff-d17cae001a0125bd53a0f8ff682e933eR4

Reply:
https://github.com/plotly/dash-table/pull/637/commits/9e46f0df6faf9e51e4373793a8400ec927549af8